addons/account/models/account_reconcile_model.py

```diff
@@ -296,7 +296,7 @@ def _get_taxes_move_lines_dict(self, tax, base_line_dict):
 
             new_aml_dicts.append({
                 'account_id': tax_res['account_id'] or base_line_dict['account_id'],
-                'name': tax_res['name'],
+                'name': ' '.join(filter(None, [base_line_dict.get('name'), tax_res['name']])),
```

Comment:
Filter None, but if it is "", it is not filtered...  A bit an unusual way to do things (needed to look it up)

Reply:
"" is fine:
```
>>> "-".join(filter(None, ["", "yay"]))
'yay'
```

Reply:
You keep persisting in weirdness then: 
`"-".join(["", "ay"])
'-ay'
`

Reply:
and None != ''

Reply:
I do not get your point. with `"-".join(filter(None, ...`, empty strings are filtered out. So it works. I get that `None != ""`, but the filter method with None as 1st param filters empty strings out. So no problems. No ? 🤔

Reply:
You are right that it will work.

Reply:
`Get move.lines dict (to be passed to the create())` you can't do that here... this is crappy

Reply:
changed